runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsConfig.java

```diff
@@ -32,6 +32,7 @@
 public interface StatusBarsConfig extends Config
 {
 	@ConfigItem(
+		position = 1,
```

Comment:
Remove the position parameters from the naturally-sorted configurations. They will be placed before configuration items with positive position values.